rb/sig/gems/open3/open3.rbs

```diff
@@ -0,0 +1,3 @@
+module Open3
```

Comment:
Do we have to commit all the gems' signatures to the repository? I wonder if we can just pull them up in the initialization phase.

Reply:
This is possible, but the reason why I did this was because gems such as:

https://github.com/ruby/open3

Do not have RBS signatures, so I could add the whole signature for the gems we need https://github.com/ruby/gem_rbs_collection, and it will be nice for the community but it will also delayed the PR because it will be the same type of work here, but for all the dependent gems

So I just did the minimum to get unblocked by other gems lacking signatures